app/models/app.rb

```diff
@@ -58,6 +58,34 @@ def maintenance
     redis.get(maintenance_key)
   end
 
+  def whitelist
+    redis.smembers(whitelist_key) || []
+  end
+
+  def whitelist=(domains)
+    whitelist.each { |domain| remove_domain(domain) }
```

Comment:
I think i wrote that code the lazy way first, then added auditing later and didn't think about the effect that was having on the audit logs. If you want to clean that up while you're in there, that'd be great.

Redis set notation would work, but you want to capture what changed on the ruby side for the auditing anyway.